internal/server/db/generate/file/write.go

```diff
@@ -28,6 +28,7 @@ func Reset(path string, imports []string, buildComment string, iface bool) error
 	content := fmt.Sprintf(`%spackage %s
 
 // The code below was generated by %s - DO NOT EDIT!
+//lint:file-ignore U1000,SA4006 Ignore staticcheck
```

Comment:
So just because the files are auto-generated doesn't mean that issues/lints aren't worth fixing.

It just means that rather than fixing the issues directly in the `.mapper.go` files, those issues need to instead be fixed in the generator code.